docs/DeveloperGuide.md

```diff
@@ -576,6 +576,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
+**Use case: UC11 - Remove a musician from a band**
+
+**MSS**
+
+1. User enters the band details and musician details.
+2. User requests to remove the musician from the band.
+3. BandConnect++ removes the musician from the band.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. User enters input wrongly
```

Comment:
Perhaps could rephrase this sentence since its unclear whether is should fall in step 1 or step 2 (kinda grey area)

Reply:
Thanks for pointing this out! Indeed, this could be described from the system's POV (because this could mean that the user enters the wrong input _after_ they request to remove the musician, which is rather ambiguous).

In a similar vein, I will also consider rephrasing step 2 to "User _attempts_ to remove ...", because it is not certain that the request to remove the musician is successful.